src/frontend/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
```

Comment:
I can see that we can provide a `mode` config here, https://vitejs.dev/config/

do we need to provide an env flag here (prod v. dev) or does `vite build` execute as production by default? Sorry, might be a noob question.

Reply:
vite automatically execute as production by default depending on the  setting within vite.config.js.